packages/inference/src/providers/hf-inference.ts

```diff
@@ -303,7 +303,10 @@ export class HFInferenceImageSegmentationTask extends HFInferenceTask implements
 	override async getResponse(response: ImageSegmentationOutput): Promise<ImageSegmentationOutput> {
 		if (
 			Array.isArray(response) &&
-			response.every((x) => typeof x.label === "string" && typeof x.mask === "string" && typeof x.score === "number")
+			response.every(
+				(x) =>
+					typeof x.label === "string" && typeof x.mask === "string" && (!("score" in x) || typeof x.score === "number")
```

Comment:
```suggestion
					typeof x.label === "string" && typeof x.mask === "string" && (x.score === undefined || typeof x.score === "number")
```

i think you can just do this (more TS-idiomatic)

Reply:
Can we just ignore the type checks on `score`?

Reply:
ah my bad I changed it and erased the change by mistake (push force)
-> done: removed the whole type check on score, as we prefer

Reply:
here the type is `score?: number;` it's a valid type, we don't want random objects or string, just an optional number